### DISCUSSION TITLE
Git submodules

### DISCUSSION THREAD

**Discussion by scivision** [scivision/mumps-superbuild#71] [General]
In the next few weeks, this MUMPS CMake project is going to use GIt submodules for the optional Lapack and Scalapack source directories for clarity and simplicity of code.

**Comment by adam-sim-dev** [comment]
Hi @scivision, I think a single git repository is easier to use.

**Comment by scivision** [comment]
I should have clarified to say this is for the optional Lapack and Scalapack source directories. It will not be required to build mumps if you already have lapack and or scalapack
